compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/SnapshotInvalidationTracker.skiko.kt

```diff
@@ -54,9 +60,6 @@ internal class SnapshotInvalidationTracker(
     }
 
     fun onDraw() {
```

Comment:
It seems there is no need to separate `onLayout` and `onDraw` anymore

Reply:
I see the reasoning, but I'm not 100% sure it's correct. I think it's possible for the behavior to be different with two flags vs. one.

But also, I don't think my changes here caused this, so I'd rather not bundle risky unrelated changes in this PR.